oneview_redfish_toolkit/api/manager.py

```diff
@@ -47,10 +47,10 @@ def __init__(self, oneview_resource, firmware_version):
         self.redfish["Description"] = None
         self.redfish["FirmwareVersion"] = firmware_version
         self.redfish["Status"] = collections.OrderedDict()
-        self.redfish["Status"]["State"] = \
-            status_mapping.get_redfish_state(oneview_resource["status"])
-        self.redfish["Status"]["Health"] = \
-            status_mapping.get_redfish_health(oneview_resource["status"])
+        state, health = status_mapping.\
```

Comment:
Will this code work well? I ask this because, before, we are using `oneview_resource["status"]` to get the `state` and `health`, and now the `get_redfish_server_hardware_status_struct` use  `oneview_resource["state"]` to get `state` and `oneview_resource["status"]` to get `health`.

Reply:
in the current version, manager can also be enclosure. Since this code will be changed, please just comment Status attribute..